src/admin-portal/pages/campaign/campaign-statistics/campaign-statistics.js

```diff
@@ -148,7 +148,7 @@ export function CampaignStatistics({}) {
                             window.open(`/campaign/${id}?preview=true`, "_blank");
                           }}
                         >
-                          <FontAwesomeIcon icon={faEye} />
+                          <FontAwesomeIcon icon={faEye} /> Sandbox Mode
```

Comment:
Using a button descriptive label that denotes an action or what clicking it will do is usually  more helpful.
Consider making this button label a verb. More so because this button is a call to action and not just a tool in a toolbox;